### PR TITLE
add Export method to UnitService

### PR DESCRIPTION
**Pull request by rkusa**
Add `dcs.unit.UnitService/Export`. For more details see inline comments.

### CODE REVIEW

File: lua/grpc.lua [hunk 1]
```diff
@@ -163,19 +163,6 @@ GRPC.errorPermissionDenied = function(msg)
...
---
-
-GRPC.toLatLonPosition = function(pos)
-  local lat, lon, alt = coord.LOtoLL(pos)
-  return {
-    lat = lat,
-    lon = lon,
-    alt = alt,
-  }
-end
```
Comment (rkusa) [inline lua/grpc.lua:177]: Moved to other exporters.

File: protos/dcs/unit/unit.proto [hunk 2]
```diff
@@ -20,6 +20,11 @@ service UnitService {
 
   // https://wiki.hoggitworld.com/view/DCS_func_getByName
   rpc Get(GetRequest) returns (GetResponse) {}
+
+  /**
+   * Get information about the unit in 3D space, including its position, orientation and velocity.
+   */
+  rpc Export(ExportRequest) returns (ExportResponse) {}
```
Comment (rkusa) [inline protos/dcs/unit/unit.proto:27]: The naming is still up for discussion. I don't think that the API is too specialized anymore to my LSO use-case as it contains all the data necessary to export any unit to e.g. a ACMI (Tacview) file.

Other naming idea: `GetTransform`

File: protos/dcs/unit/unit.proto [hunk 3]
```diff
@@ -39,6 +44,44 @@ message GetPositionResponse {
...
+  double v = 3;
+  // The heading of the object on a flat world. To get the heading corrected by the projection error
+  // when going from a flat to a spherical world, use `orientation.yaw` instead.
+  double heading = 4;
+  Orientation orientation = 5;
+  // The direction the object is traveling to, and speed (magnitude of the vector) the object is
+  // traveling with.
+  dcs.common.Vector velocity = 6;
+  double time = 7; // Time in seconds since the scenario started.
+}
```
Comment (rkusa) [inline protos/dcs/unit/unit.proto:83]: yaw, pitch, roll, u, v and heading map what TacView is using (both in naming and semantic)

File: protos/dcs/unit/unit.proto (hunk 2 above)
Comment (rurounijones) [reply protos/dcs/unit/unit.proto:27]: Yeah, I am not sure on the `Export` name, it is very... generic. `GetTransform` seems to be the name for this kind of stuff in graphics and physics libraries so shall we go with that?

**Comment by rkusa** [comment]
Rebased due to merge conflicts, and renamed `Export` to `GetTransform`

**Review by rurounijones — APPROVED** [review]

**Pull request merged**